app/scripts/app.js

```diff
@@ -47,7 +47,8 @@ app.config(function($locationProvider, $stateProvider, $urlRouterProvider, confi
                     var filters = {
                         keyword: $stateParams.keyword,
                         organization: $stateParams.organization,
-                        state: $stateParams.state
+                        state: $stateParams.state,
+                        status: $stateParams.status
```

Comment:
I don't like status either for the same reason than you, but if you look through the code the "color" is always referred to by "status", so I preferred stay consistent.